policies/policies/romeo_112.yml

```diff
@@ -24,40 +24,40 @@ parent-policies: []
 child-policies: []
 
 # Leave any notes about upstream RoMEO issues here. For example, if this policy does not apply to the journals listed above, leave a note here.
-flag-romeo:
+flag-romeo: there are different peer review urls for each journal - using PLOS ONE below
 
 # Peer review policy url (valid url)
-peer-review-url:
+peer-review-url: http://journals.plos.org/plosone/s/editorial-and-peer-review-process
```

Comment:
I think the build is failing because the YAML is incorrect. Strings containing `:` need to be quoted I think. Try quotes around URLs like:

```yml
peer-review-url: "http://journals.plos.org/plosone/s/editorial-and-peer-review-process"
```

Perhaps a YAML linter could help us detect these formatting problems.

Reply:
According to https://github.com/adrienverge/yamllint/issues/30#issuecomment-265976399 the colon in the URL may actually be okay, but let's see if the issues are resolved when we quote URLs.